blocks/newsletter-modal/newsletter-modal.css

```
@@ -0,0 +1,150 @@
+body.newsletter-no-scroll {
+  overflow: hidden;
+}
+
+.newsletter-modal {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: var(--color-white);
+  overflow: hidden;
+}
+
+.newsletter-modal-banner-container {
+  width: 100%;
+}
+
+.newsletter-modal-banner-container img {
+  width: 100%;
+}
```

Would you want to consolidate these?
```suggestion
.newsletter-modal-banner-container,
.newsletter-modal-banner-container img {
  width: 100%;
}
```